src/lib/Commands/InitCommand.js

```diff
@@ -119,10 +143,10 @@ class InitCommand extends AbstractCommand {
     for (const modelId in this._models) {
       for (const deviceId of  this._models[modelId].devices) {
         table.push([
-          deviceId,
-          this._devices[deviceId].name,
-          this._models[modelId].id,
-          this._models[modelId].name,
+          deviceId || "",
```

Comment:
This is change has been made In the electricimp/impTest develop branch: https://github.com/electricimp/impTest/commit/b64e9c4b64f0b533015c382ddc70653e23aa065d